dandi/girder.py

```diff
@@ -556,6 +556,71 @@ def lock_dandiset(self, dandiset_identifier: str):
                         f"Failed to unlock dandiset {dandiset_identifier} due to: {msg}"
                     )
 
+    NGINX_MAX_CHUNK_SIZE = 100 * (1 << 20)  # 100 MiB
```

Comment:
```suggestion
    NGINX_MAX_CHUNK_SIZE = 400 * (1 << 20)  # 100 MiB
```
@mgrauer raised up the limits to 400, so I will suggest and accept right away to proceed

Reply:
You should probably update the comment as well.

Reply:
yeap yeap -- I did